### PR TITLE
Update example application

### PR DESCRIPTION
**Pull request by cliuamzn**
*Issue:*
- GitHub project requires security patching

*Description of changes:*
- Code refactor
- Updated versions of dependencies in requirements.txt

*Test:*
- Manually test with local DynamoDB and SNS

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -1,53 +1,52 @@
...
+    'THEME') is None else os.environ.get('THEME')
+FLASK_DEBUG = 'false' if os.environ.get(
+    'FLASK_DEBUG') is None else os.environ.get('FLASK_DEBUG')
 
 # Create the Flask app
 application = flask.Flask(__name__)
 
 # Load config values specified above
-application.config.from_object(__name__)
+application.config.from_object("default_config")
```
Comment (garrettjaeger) [inline application.py:33]: We should be consistent about using single quotes and double quotes

File: application.py [hunk 2]
```diff
@@ -1,53 +1,52 @@
...
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-import sys
 import json
+import logging
+import os
 
+import boto3
```
Comment (garrettjaeger) [inline application.py:19]: Should we import specific items from boto instead of importing the whole package?

File: application.py (hunk 1 above)
Comment (cliuamzn) [reply application.py:33]: Yes. Will update this part

File: application.py (hunk 2 above)
Comment (cliuamzn) [reply application.py:19]: Sounds good. Will just add required classes

**Review by tommypraeger — APPROVED** [review]

**Pull request merged**